site/blog/2022/deploy-static-website.mdx

```diff
@@ -0,0 +1,193 @@
+---
+slug: deploy-static-website
+title: Deploy a Static Website Using SQL
+date: 2022-12-21
+unlisted: true
+authors: [mtp1376]
+---
+Did you know you can deploy a static website without leaving your SQL REPL? In this post we'll show you how to leverage IaSQL's capabilities to simply deploy a static website from your GitHub repository to AWS using only SQL commands.
+
+## Overview
+First, we will create a S3 bucket which will hold our static files. Then we'll enable S3 static website hosting to be able to access our website using a S3 endpoint. We'll also need to enable public access to our S3 files. After that we will create a new CloudFront distribution because S3 static website hosting doesn't support HTTPS, and we're done. Let's dive in to see how IaSQL can deploy a static website using simple SQL commands.
```

Comment:
The process explained here does not seem that simple and straightforward to me, seems like a lot of steps. Maybe just say, "Create and configure a new s3 bucket to host your website. If you want HTTPS enabled let's configure a new CloudFront distribution and we are done" (just paraphrasing).